### ISSUE TITLE
if present docker image with psql timescaledb support?

### ISSUE THREAD

**Issue by kuznetcoff777** [zabbix/zabbix-docker#907]
Hi! 
I dont see an option with timesacedb in compose files. Why is that?

**Comment by dotneft** [comment]
It is optional. Feel free to add this one. Just replace postgresql image and enable timescale db support for zabbix db.

**Comment by kuznetcoff777** [comment]
Sorry, another question, about high availablility, if it is possible to enable this option?

**Comment by dotneft** [comment]
high availability for what (DB, Zabbix components)?

**Comment by kuznetcoff777** [comment]
Native HA for zabbix-server

**Comment by dotneft** [comment]
feel free to use it with trunk version:
```
ZBX_AUTOHANODENAME=fqdn # Allowed values: fqdn, hostname. Available since 6.0.0
ZBX_HANODENAME= # Available since 6.0.0
ZBX_AUTONODEADDRESS=fqdn # Allowed values: fqdn, hostname. Available since 6.0.0
ZBX_NODEADDRESSPORT=10051 # Allowed to use with ZBX_AUTONODEADDRESS variable only. Available since 6.0.0
ZBX_NODEADDRESS=localhost # Available since 6.0.0
```
